src/main/java/Ultron/Tasks/Todo.java

```diff
@@ -0,0 +1,57 @@
+package ultron.tasks;
+
+public final class Todo extends Task {
```

Comment:
Is there a need to declare Todo as a final class? I think there are still some possibilities to extend this class though we might not need it in our iP

Reply:
When the extension comes, I can just remove the final after that and extend from there?
What I was thinking was that everything extends from the abstract class Task.
If there are other similar classes as todo we can all group them under todo or modify todo to accommodate other functions